aws-aurora-mysql/variables.tf

```
@@ -167,3 +167,9 @@ variable "engine_version" {
   type    = string
   default = "5.7"
 }
+
+variable ca_cert_identifier {
+  type        = string
+  description = "Identifier for the certificate authority. Use rds-ca-2015 for anything new."
```

```suggestion
  description = "Identifier for the certificate authority. rds-ca-2015 is the latest available version."
```
not sure how I feel about that change, but maybe?